libs/proxy/postgres-protocol2/src/message/backend.rs

```diff
@@ -541,6 +541,10 @@ impl NoticeResponseBody {
     pub fn fields(&self) -> ErrorFields<'_> {
         ErrorFields { buf: &self.storage }
     }
+
+    pub fn raw(&self) -> Bytes {
```

Comment:
Can this return a byte slice and be called as_bytes? BeMessage doesn't need a clone, right?